bridges/snowbridge/pallets/outbound-queue-v2/src/lib.rs

```diff
@@ -122,7 +122,15 @@ pub mod pallet {
 
 		type Hashing: Hash<Output = H256>;
 
-		type MessageQueue: EnqueueMessage<AggregateMessageOrigin>;
+		type AggregateMessageOrigin: FullCodec
+			+ MaxEncodedLen
+			+ Clone
+			+ Eq
+			+ PartialEq
+			+ TypeInfo
+			+ Debug;
+		type OriginToAggregateMessageOrigin: Convert<H256, Self::AggregateMessageOrigin>;
```

Comment:
Why not use the same name? Bellow you have for the same: `type AggregateMessageOriginConverter: ConvertBack<ParaId, Self::AggregateMessageOrigin>;`

```suggestion
		type AggregateMessageOriginConverter: Convert<H256, Self::AggregateMessageOrigin>;
```

Reply:
`OriginToAggregateMessageOrigin` is confused. We can use `AggregateMessageOriginConverter` as @bkontur suggested.